src/presenter.rs

```diff
@@ -348,10 +353,17 @@ impl<'a> Presenter<'a> {
             other => self.state = other,
         }
     }
+
+    fn suspend(&self, drawer: &mut TerminalDrawer<Stdout>) {
+        drawer.terminal.suspend();
+        unsafe { libc::raise(libc::SIGTSTP) };
```

Comment:
I imagine this won't work on windows? If that's the case this needs a `#[cfg(...)]` guard so it's a no op in that platform.

Reply:
Ah, good point!